Add find_package Threads otherwise cmake 3.30 complains about missing
Threads:Threads target
Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -9,6 +9,8 @@ endif ()
 
 project(Zeek C CXX)
 
+find_package(Threads)
+
 # We want to set ENABLE_DEBUG to ON by default if the build type is Debug.
 set(ENABLE_DEBUG_DEFAULT OFF)
 if (NOT GENERATOR_IS_MULTI_CONFIG)
@@ -487,7 +489,7 @@ if (NOT ZEEK_ETC_INSTALL_DIR)
 endif ()
 
 if (NOT ZEEK_STATE_DIR)
-    set(ZEEK_STATE_DIR ${ZEEK_ROOT_DIR}/var/lib)
+    set(ZEEK_STATE_DIR /var/db)
 endif ()
 
 if (NOT ZEEK_SPOOL_DIR)
@@ -1219,7 +1221,7 @@ if (INSTALL_ZKG)
     configure_file(${CMAKE_CURRENT_SOURCE_DIR}/zkg-config.in ${CMAKE_CURRENT_BINARY_DIR}/zkg-config
                    @ONLY)
 
-    install(DIRECTORY DESTINATION var/lib/zkg)
+    install(DIRECTORY DESTINATION var/db/zkg)
     install(FILES ${CMAKE_CURRENT_BINARY_DIR}/zkg-config DESTINATION ${ZEEK_ZKG_CONFIG_DIR}
             RENAME config)
 endif ()
